<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMDDialog</class>
 <widget class="QDialog" name="CMDDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>829</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>0</y>
     <width>491</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Courier</family>
    </font>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0004;&quot;&gt;Remember!!&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; color:#ff0004;&quot;&gt;Make sure the green light is on the top of the &amp;quot;Ref Enable&amp;quot;.&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;HomePos, WalkReady : Hubo takes the basic posture. &lt;/p&gt;&lt;p&gt;Be careful because this motion does not consider selfcollision.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>190</y>
     <width>16</width>
     <height>441</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>20</y>
     <width>261</width>
     <height>101</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="BTN_HOMEPOS">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>HomePos</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="BTN_WALKREADY">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>WalkReady</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>210</y>
     <width>301</width>
     <height>102</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;Gripper Command&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QRadioButton" name="RB_BOTH">
          <property name="text">
           <string>Both Hand</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">GRB_GRIPPER</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="RB_RH">
          <property name="text">
           <string>Right Hand</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">GRB_GRIPPER</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="RB_LH">
          <property name="text">
           <string>Left Hand</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">GRB_GRIPPER</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="BTN_GRIPPER_OPEN">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>OPEN</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="BTN_GRIPPER_STOP">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>STOP</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="BTN_GRIPPER_CLOSE">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>CLOSE</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>210</y>
     <width>241</width>
     <height>411</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;JoyStick&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <property name="spacing">
       <number>0</number>
      </property>
      <item>
       <widget class="QTableWidget" name="JOY_TABLE_INFO_LEFT">
        <property name="font">
         <font>
          <pointsize>7</pointsize>
         </font>
        </property>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>8</number>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>27</number>
        </attribute>
        <attribute name="verticalHeaderMinimumSectionSize">
         <number>10</number>
        </attribute>
        <row>
         <property name="text">
          <string>L-LT</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>L-LB</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>L-JOG-RL</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>L-JOG-UD</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>L-ARW_RL</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>L-ARW_UD</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>R-JOG-BTN</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>L-JOG-BTN</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>BTN-BACK</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>Value</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <widget class="QTableWidget" name="JOY_TABLE_INFO_RIGHT">
        <property name="font">
         <font>
          <pointsize>7</pointsize>
         </font>
        </property>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>8</number>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>27</number>
        </attribute>
        <attribute name="verticalHeaderMinimumSectionSize">
         <number>10</number>
        </attribute>
        <row>
         <property name="text">
          <string>R-RT</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>R-RB</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>R-JOG-RL</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>R-JOG-UD</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>BTN-Y</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>BTN-X</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>BTN-B</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>BTN-A</string>
         </property>
        </row>
        <row>
         <property name="text">
          <string>BTN-START</string>
         </property>
        </row>
        <column>
         <property name="text">
          <string>Value</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QPushButton" name="BTN_JOYSTART">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Joy Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BTN_JOYSTOP">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>Joy Stop</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="autoDefault">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QPushButton" name="BTN_JOYMOVE">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>JoyMove Start</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="BTN_MOVESTOP">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="text">
         <string>JoyMove Stop</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>label_2</zorder>
  <zorder>line</zorder>
  <zorder>label_3</zorder>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="GRB_GRIPPER"/>
 </buttongroups>
</ui>
